packages/office-addin-usage-data/src/defaults.ts

```diff
@@ -6,4 +6,13 @@ import * as path from "path";
 
 export const usageDataJsonFilePath: string = path.join(os.homedir(), "/office-addin-usage-data.json");
 export const groupName = "office-addin-usage-data";
-export const instrumentationKeyForOfficeAddinCLITools = "de0d9e7c-1f46-4552-bc21-4e43e489a015";
\ No newline at end of file
+export const instrumentationKeyForOfficeAddinCLITools = "de0d9e7c-1f46-4552-bc21-4e43e489a015";
+// Office-Addin-Scripts packages currently sending usage data
+export const officeAddinScriptsPackages: string[] = [
```

Comment:
I'm concerned about this pattern where these package names are specified. Isn't there a better way?

Reply:
Yeah I am not crazy about this design either. The other option is to update all the consuming office-addin-script packages to not export an usage data object and to just create it at runtime. But this opens up the clunky pattern of creating the usage data object multiple times in each file that sends usage data.  I am really striving for as minimal changes as possible, but if you thinking creating the object at runtime is the best approach, we can do that

Reply:
I think a chat might help us discuss and come up with a better proposal to deal with this.